<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-28T12:48:49 PST" name="Default suite" finished-at="2022-11-28T12:49:03 PST" duration-ms="14409">
    <groups>
    </groups>
    <test started-at="2022-11-28T12:48:49 PST" name="Default test" finished-at="2022-11-28T12:49:03 PST" duration-ms="14409">
      <class name="tests.LoginSteps_Simple_testngAnno">
        <test-method is-config="true" signature="user_is_on_test_environment_page()[pri:0, instance:tests.LoginSteps_Simple_testngAnno@c1fca1e]" started-at="2022-11-28T12:48:49 PST" name="user_is_on_test_environment_page" finished-at="2022-11-28T12:48:58 PST" duration-ms="9613" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- user_is_on_test_environment_page -->
        <test-method signature="loginTestPage()[pri:1, instance:tests.LoginSteps_Simple_testngAnno@c1fca1e]" started-at="2022-11-28T12:48:58 PST" name="loginTestPage" description="loginTestPage , priority one" finished-at="2022-11-28T12:49:03 PST" duration-ms="4312" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [My account � Top Tech School] but found [My account – Top Tech School]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [My account � Top Tech School] but found [My account – Top Tech School]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at tests.LoginSteps_Simple_testngAnno.loginTestPage(LoginSteps_Simple_testngAnno.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestPage -->
        <test-method signature="schoolAccount()[pri:2, instance:tests.LoginSteps_Simple_testngAnno@c1fca1e]" started-at="2022-11-28T12:49:03 PST" name="schoolAccount" description="schoolAccount, priority two" finished-at="2022-11-28T12:49:03 PST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- schoolAccount -->
        <test-method signature="homeAccount()[pri:3, instance:tests.LoginSteps_Simple_testngAnno@c1fca1e]" started-at="2022-11-28T12:49:03 PST" name="homeAccount" description="homeAccount, priority three" finished-at="2022-11-28T12:49:03 PST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homeAccount -->
        <test-method is-config="true" signature="gracefulTearDown()[pri:0, instance:tests.LoginSteps_Simple_testngAnno@c1fca1e]" started-at="2022-11-28T12:49:03 PST" name="gracefulTearDown" finished-at="2022-11-28T12:49:03 PST" duration-ms="394" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gracefulTearDown -->
      </class> <!-- tests.LoginSteps_Simple_testngAnno -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
